### PR TITLE
Add a new module to display charge status of a Tesla

### PR DESCRIPTION
**Pull request by drwahl**
This module will use the Tesla API to display the charge information for your Tesla vehicle.

### CODE REVIEW

**Comment by terminalmage** [comment]
This breaks CI because Travis doesn't have the `tesla_api` module installed. It should probably be gated in some way and then an error raised if you try to _use_ the module without `tesla_api` installed. Or the import could be moved into an `init()` function.

@enkore What are your thoughts on this?

**Comment by enkore** [comment]
See https://i3pystatus.readthedocs.io/en/latest/module.html#handling-dependencies

You'd just need to add "tesla_api" to MOCK_MODULES in docs/conf.py

File: i3pystatus/teslacharge.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    datapoints that can be displayed.
+    Requires: tesla_api
+    """
+
+    settings = (
+        ("email", "Email address used to login to your Tesla account."),
+        ("password", "Password for accessing your Tesla account."),
+        ("units", "Miles (default) or km"),
+        ("interval", "Poll frequency. Polling too often can drain the battery"),
+        "format"
```
Comment (enkore) [inline i3pystatus/teslacharge.py]: You're not using this one

File: i3pystatus/teslacharge.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    units = "miles"
+    conversion_factor = 0.62137119
+    charge_complete = "#00FF00"  # green
+    charging = "#FFFF00"         # yellow
+    offline_color = "#FF0000"    # red
+    interval = 900
+    charging_icon = '⚡'
+    disconnect_icon = ''
+    format_with_charge = "{name}: {charge_state_icon}{charge_rate} {battery_level}%/{charge_limit_soc}% ({battery_range})"
+    format_without_charge = "{name}: {battery_level}%/{charge_limit_soc}% ({battery_range})"
```
Comment (enkore) [inline i3pystatus/teslacharge.py]: Only these two -- add them to `settings` above and you should be good to go

That's the cause of

> Module <TeslaCharge> has non-required setting(s) <format> with no default!

**Comment by enkore** [comment]
The future is _naow_.

**Pull request merged**